src/gluonts/model/n_beats/_estimator.py

```diff
@@ -0,0 +1,287 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# Standard library imports
+from typing import List, Optional
+
+# Third-party imports
+from mxnet.gluon import HybridBlock
+import mxnet as mx
+
+# First-party imports
+from gluonts.core.component import validated
+from gluonts.dataset.field_names import FieldName
+from gluonts.model.estimator import GluonEstimator
+from gluonts.model.predictor import Predictor, RepresentableBlockPredictor
+from gluonts.trainer import Trainer
+from gluonts.transform import (
+    Chain,
+    ExpectedNumInstanceSampler,
+    InstanceSplitter,
+    Transformation,
+)
+
+# Relative imports
+from ._network import (
+    NBEATSPredictionNetwork,
+    NBEATSTrainingNetwork,
+    VALID_N_BEATS_STACK_TYPES,
+    VALID_LOSS_FUNCTIONS,
+)
+
+
+class NBEATSEstimator(GluonEstimator):
+    """
+    An Estimator based on a single NBEATS Network. The actual NBEATS model
+    is an ensemble of NBEATS Networks.
+
+    Parameters
+    ----------
+    freq
+        Time time granularity of the data
+    prediction_length
+        Length of the prediction horizon
+    context_length
+        Number of time units that condition the predictions
+        (default: None, in which case context_length = prediction_length)
+    trainer
+        Trainer object to be used (default: Trainer())
+    num_stacks:
```

Comment:
Note that the colon is not needed (also in the other parameters)

```suggestion
    num_stacks
```

Reply:
done